scripts/provisioning/s3_prov_config.yaml

```diff
@@ -102,6 +102,7 @@ TEST:
 #Default values of optional params
 
 DEFAULT_POST_INSTALL:
+  CONFSTORE_S3_LOGGER_NAME: "s3deployment"
```

Comment:
Remove it from post install phase, this will be an Unsolicited Key.

Reply:
Also, add a key to get max log size in MB with a comment that this is for deployment and keep these 2 keys together.
S3_DEPLOYMENT_LOGGER_NAME
S3_DEPLOYMENT_LOGGER_MAX_SIZE_MB

Reply:
Done.